.cspell.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "0.1",
+  "words": [
+    "ansible"
+  ]
+}
```

Comment:
We already has https://github.com/prettier/prettier/blob/master/cspell.json

Reply:
Thanks. Fixed!

Reply:
Sorry, but you need fix indention, and remove the final newline. https://github.com/prettier/prettier/pull/6684#discussion_r336745585

Reply:
Done @fisker